examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -88,25 +81,28 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
 	}
 
 	public async loadExisting(id: string): Promise<ModelType> {
-		const container = await this.loader.resolve({
-			url: id,
-			headers: {
-				[LoaderHeader.loadMode]: {
-					// Here we use "all" to ensure we are caught up before returning.  This is particularly important
-					// for direct-link scenarios, where the user might have a direct link to a data object that was
-					// just attached (i.e. the "attach" op and the "set" of the handle into some map is in the
-					// trailing ops).  If we don't fully process those ops, the expected object won't be found.
-					opsBeforeReturn: "all",
+		const container = await resolveContainer({
+			request: {
+				url: id,
+				headers: {
+					[LoaderHeader.loadMode]: {
+						// Here we use "all" to ensure we are caught up before returning.  This is particularly important
+						// for direct-link scenarios, where the user might have a direct link to a data object that was
+						// just attached (i.e. the "attach" op and the "set" of the handle into some map is in the
+						// trailing ops).  If we don't fully process those ops, the expected object won't be found.
+						opsBeforeReturn: "all",
+					},
 				},
 			},
+			...this.props,
```

Comment:
Same here, order should be flipped such that the explicitly provided id to this method is the one that takes precedent.